test/index.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 export NODE_ENV=${NODE_ENV:-test}
+if [ "$NODE_ENV" == "betanet" -o "$NODE_ENV" == "development" -o "$NODE_ENV" == "testnet" ]; then
```

Comment:
that is not intended, tests should generally be possible to run on other environments, but might need to be adjusted

e.g. we run on `ci` and `ci-betanet` for now https://github.com/near/near-cli/blob/b3f5bde78451901d1e6cc8392e2ef2fc0bab23b9/.travis.yml#L6 but ideally we should be able to run as is on `testnet` / `betanet`

Reply:
I agree with you, but right now tests fail if NODE_ENV=betanet|development|testnet, and it's hard to see **why** tests are failing. It took me some time to see that the problem was my NODE_ENV. I added this to avoid the same pitfall for others.
I agree with you that if tests are adjusted, this warning should also be updated or removed.